README.md

```diff
@@ -43,10 +43,29 @@ If you omit `npm-token`, then packages will be prepared for publishing, but no p
 
 You can optionally send deployment announcements to Slack by providing a `slack-webhook-url` input:
 
+The absolute minimum configuration for this is:
+
+```yaml
+- uses: MetaMask/action-npm-publish@v2
+  with:
+    slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+```
+
+We've now added the ability to customize the message posted in Slack and those optional inputs are as follows:
+
+- `icon-url`: Url to the avatar used for the bot in Slack. If not set this defaults to the avatar in this repository.
+- `bot-name`: The name of the bot as it appears on Slack. If not set this defaults to `MetaMask bot`.
+- `bot-at`: Use this if you want to ping an individual or subset of individuals on Slack using @.
```

Comment:
Nit: I'm wondering if `bot-at` is as clear as it could be. What do you think about `recipient-name` or `target-name`? Also, if `bot-name` doesn't include the leading `@`, should this? i.e. Would it make sense to pass this option without a leading `@` — should the `@` be added automatically?

Reply:
> should the @ be added automatically
I was contemplating the same thing and it's easily doable. If that's your preference I am happy to do it that way. it means one less char config wise
maybe for `bot-name` we should just use `username` since that's what it maps to api side?